content/en/serverless/step_functions/_index.md

```diff
@@ -15,12 +15,14 @@ further_reading:
   text: "Serverless Monitoring for AWS Lambda"
 ---
 
-AWS Step Functions is a serverless orchestration service that lets developers create and manage multi-step application workflows. In addition to getting Cloudwatch metrics from Datadog's [AWS Step Functions integration][2], Datadog also provides native AWS Step Function monitoring and tracing.
+AWS Step Functions is a serverless orchestration service that lets developers create and manage multi-step application workflows. In addition to getting Cloudwatch metrics from Datadog's [AWS Step Functions integration][2], Datadog also provides AWS Step Function tracing, logs, and [enhanced metrics][3] through the collection of Cloudwatch logs.
 
 {{< img src="serverless/step_functions/overview1.png" alt="An AWS Step Function oveview tab." style="width:100%;" >}}
 
 ### How it works
-Datadog AWS Step Functions Monitoring collects logs and integration metrics from the AWS integration and uses ingested logs from AWS Step Functions to generate enhanced metrics and traces for your Step Function executions.
+Datadog AWS Step Functions Monitoring makes use of Cloudwatch metrics from the [AWS Step Functions integration][2] and Cloudwatch logs sent through the Datadog Forwarder or Amazon Data Firehose. The latter provides tracing and [enhanced metrics][3].
+
+{{< img src="serverless/step_functions/how_it_works.png" alt="Diagram showing two components of Datadog AWS Step Function monitoring: Cloudwatch metrics sent through the AWS Step Functions integration, and logs, traces, and enhanced metrics sent through the Datadog Lambda Forwarder or Amazon Data Firehose." style="width:100%;" >}}
```

Comment:
The workflow is:
1) Logs are sent through the Datadog Lambda Forwarder or Amazon Data Firehose to Datadog. Both Forwarder and Firehose are running in customer's environment.
2) In Datadog, our internal service converts logs to traces and enhanced metrics. 
I just synced with @jakegreenberg1, and he will update the "how_it_works".png file accordingly.